runtime/src/error.c

```diff
@@ -79,6 +79,11 @@ static void spinhaltIfAlreadyExiting(void) {
 #endif
 }
 
+#define UNW_LOCAL_ONLY
+#include<libunwind.h>
+extern c_string chpl_funSymTable[];
+extern int32_t chpl_sizeFunSymTable;
```

Comment:
These two declarations belong in runtime/include/chplcgfns.h.
Additionally, chpl_sizeFunSymTable isn't strictly necessary since you could do sizeof(chpl_funSymTable) / (4*sizeof(c_string)) but that is a minor nit. I think I'd still rather have a different array for the integer values (line numbers) though.